src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    autocomplete="off" onsubmit="onSubmit()">
+      <div>
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="entr">
+            <label for="sur">
+              Surname: <input type="text" name="surname"
+              id="sur" maxlength="10">
```

Comment:
Each tag should be on new line.
About attributes - see checklist item 3.
Also, id and name can be the same. Don't use shortened words for id.
Fix the attributes and indentations all over your project.
```suggestion
              Surname:
              <input
                type="text"
                name="surname"
                id="surname"
                maxlength="10"
              >
```